src/cms/templates/push_notifications/push_notification_list.html

```diff
@@ -29,7 +29,7 @@
                 <th class="text-sm text-left uppercase py-3 px-2">
                     <div class="lang-grid flags">
                         {% for lang in languages %}
-                        <a href="{% url 'push_notifications' region_slug=region.slug language_code=lang.code %}">
+                        <a href="{% url 'push_notifications' region_slug=region.slug language_slug=lang.code %}">
```

Comment:
```suggestion
                        <a href="{% url 'push_notifications' region_slug=region.slug language_slug=lang.slug %}">
```
Just one appearance where code is still wrong... maybe there are others, too?